tests/components/gios/test_sensor.py

```diff
@@ -50,7 +51,7 @@ async def test_sensor(hass: HomeAssistant) -> None:
         == CONCENTRATION_MICROGRAMS_PER_CUBIC_METER
     )
     assert state.attributes.get(ATTR_ICON) == "mdi:molecule"
-    assert state.attributes.get(ATTR_INDEX) == "bardzo dobry"
+    assert state.attributes.get(ATTR_INDEX) == "very_good"
```

Comment:
You can also provide translations for attributes in our strings.json file.

<https://developers.home-assistant.io/docs/internationalization/core#entity-state-attributes>

Although the frontend currently only supports climate right now, it would be a matter of time before the rest is implemented. The backend supports it fully.

Reply:
I plan to migrate `index` attributes to the separate entities in the next step. And remove `name` and `station` attributes because they present static data.

Reply:
I'm always all in favor of less attributes 🎉 😬